gps2val/readme.md

```diff
@@ -2,10 +2,8 @@
 consumes [gpsd](https://gpsd.gitlab.io/gpsd/) as datasource and pushes location to kuksa.val server.
 The [`gpsd_feeder.ini`](./config/gpsd_feeder.ini) contains `kuksa.val` and `gpsd` configuration.
 
-Before starting the gps feeder, you need start `kuksa.val` and `gpsd`:
+Before starting the gps feeder, you need to start `kuksa.val server/databroker` (for this have a look at [KUKSA.val server](../../kuksa-val-server/README.md) and [KUKSA.val databroker](../../kuksa_databroker/README.md)). You have to start an instance of `gpsd` by running:
```

Comment:
Maybe use absolute links, because if you did not check this out as submodule (or just clicking in Github Webif) you can not reach the doc, so I suggest
../../kuksa-val-server/README.md -> https://github.com/eclipse/kuksa.val/blob/master/kuksa-val-server/README.md etc.